src/components/FeatureTrainingModal.tsx

```diff
@@ -140,6 +143,7 @@ function FeatureTrainingModal({
     contentInnerContainerStyles,
     contentOuterContainerStyles,
     modalInnerContainerStyle,
+    isVisible: isVisibleProp = true,
```

Comment:
Just for my understanding, why can't we use the `isVisible` directly, defaulted to true? Why do we need to assign it to this new `isVisibleProp` variable?

Reply:
What's the difference between isModalVisible and isVisible and isVisibleProp?

Reply:
Interesting. Is there a different variable name we could use to potentially better differentiate isModalVisible and isVisible. So it's more obvious from the names what each are used for?

Reply:
That sounds good to me. It feels more differentiated.